civiclens/templates/search_results.html

```diff
@@ -33,28 +33,20 @@
             <!-- sort section -->
             <span class="h6 font-weight-bold" >Sort by:</span>
             <div class="form-check">
-              <input class="form-check-input" type="radio" name="sort_by" id="most_relevant" value="most_relevant">
```

Comment:
note about the merge here - the ADA compliance requires titles for inputs, so when you see those pop up in the version on main, keep them but feel free to change the names etc to make them make sense

Reply:
will do!

Reply:
Just flagged that my code matches main in slack, and the possibility that an earlier push from myself removed these